bats/fb-test-katello.bats

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env bats
```

Comment:
@evgeni I think one additional check could be valuable here -- doing a curl of `/katello/api/ping` since this provides information of about services that the status check does not, e.g. whether or not Katello can talk to Pulp or Candlepin and whether or not Pulp is having any connectivity issues with the workers. These are things that can produce big smells for installation health.